src/SecretsFoundry.ts

```diff
@@ -125,3 +129,5 @@ export class SecretsFoundry {
     throw new Error(`No loader exists for: ${value}`);
   }
 }
+
+export const DELIMITER = "->";
```

Comment:
nit: this might be better somewhere else, like Utils?

Reply:
Addressed